tests/setup/mod.rs

```diff
@@ -275,6 +273,26 @@ pub mod nigiri {
         Ok(())
     }
 
+    fn nigiri_is_running_in_non_ci_mode() -> bool {
+        nigiri_is_running() && !nigiri_runs_in_ci_mode()
+    }
+
+    fn nigiri_is_running() -> bool {
+        exec(vec!["jq", "-r", ".running", &get_nigiri_config_path()])
+            .stdout
+            .starts_with(b"true")
+    }
+
+    fn nigiri_runs_in_ci_mode() -> bool {
+        exec(vec!["jq", "-r", ".ci", &get_nigiri_config_path()])
+            .stdout
+            .starts_with(b"true")
+    }
+
+    fn get_nigiri_config_path() -> String {
+        env::var("HOME").unwrap() + "/.nigiri/nigiri.config.json"
```

Comment:
This assumes the default nigiri path for Linux. On macOS, the default is different -> `$HOME/Library/Application\ Support/Nigiri`

Reply:
Becomes obsolete with: 5ea7f513005ffef4fb557f364aeae2d80e5f0fda